bookstore_api/urls.py

```diff
@@ -1,10 +1,20 @@
 from django.urls import path
-from bookstore_api import views, users
- 
-urlpatterns = [ 
+from bookstore_api import views
+from .views import GenericAPIView
```

Comment:
these imports and a few lines of codes need to be deleted. They give you errors and I remembered we already resolved it but not sure why the old codes still showed up #

Reply:
Agreed with houy, we should not delete ",users" and urlpatterns link must remain